int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -280,6 +280,11 @@ function countryAlpha3(incomingCountryIso2) {
 	countryDict.forEach((val) => iso2ToIso3CountryDict = (incomingCountryIso2.match(val.key)) ? val.value : iso2ToIso3CountryDict);
 	return iso2ToIso3CountryDict || vDefaultCountry;
 }
+function countryAlpha2(incomingCountryIso3) {
+	var iso3ToIso2CountryDict;
+	countryDict.forEach((val) => iso3ToIso2CountryDict = (incomingCountryIso3.match(val.value)) ? val.key : iso3ToIso2CountryDict);
+	return iso3ToIso2CountryDict;
+}
```

Comment:
This is a little confusing. Your code obfuscates it's purpose which is to simply transform the ISO3 country to the ISO2 country. 

```javascript

function countryAlpha2(incomingCountryIso3) {
  return countryDict.filter(function(countryKeyAndValue) { 
    // incomingCountryIso3, e.g. USA is equal to the value in a countryDict, e.g. { key: "US", value: "USA" }
    return countryKeyAndValue.value === incomingCountryIso3;
  })[0] || null; 
}

```

Reply:
Changed this function to match your example.